src/test/resources/com/xceptance/neodymium/testclasses/data/FixedRandomnessOfDataSets.csv

```diff
@@ -0,0 +1,5 @@
+key1
+val1
+val2
+val3
+val4
```

Comment:
Please add a newline add the end of every new file.

Reply:
Done